src/EditorFeatures/Core/Implementation/CodeDefinitionWindow/ICodeDefinitionWindowService.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.CodeDefinitionWindow
+{
+    internal interface ICodeDefinitionWindowService
+    {
+        Task SetContextAsync(ImmutableArray<CodeDefinitionWindowLocation> locations, CancellationToken cancellationToken);
```

Comment:
what does it mean to report two locations to the codedef window?  does it handle that somehow?  if so, pix please?

Reply:
So I hadn't tried this and it works, but badly. If you right click in the window, you can jump between multiple definitions:

![image](https://user-images.githubusercontent.com/201340/132428215-39d2450a-d51c-49fb-a7c7-dd7230f4790f.png)

Why there's not just buttons I can't say...

Reply:
would love to see telemetry showing if these are ever used ;0

Reply:
Telemetry says 100% of the users that use C#'s code definition window have used these buttons. :smile: